src/components/Organisms/AlertManager/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { useAlertHandlerContext } from "../../../contexts/alert_handler";
+import Alert from "../../Atoms/Alert";
+import React, { useEffect } from "react";
+
+const AlertManager = () => {
+  const AlertHandlerContext = useAlertHandlerContext();
+  useEffect(() => {
+    if (AlertHandlerContext.visible === true) {
+      setTimeout(() => {
+        AlertHandlerContext.closeAlert();
+      }, 5000);
+    }
+  }, [AlertHandlerContext]);
+  return (
+    <Alert
+      alertMessage={AlertHandlerContext.alertMessage}
+      visible={AlertHandlerContext.visible}
```

Comment:
```suggestion
  const alertHandlerContext = useAlertHandlerContext();
  useEffect(() => {
    if (alertHandlerContext.visible === true) {
      setTimeout(() => {
        alertHandlerContext.closeAlert();
      }, 5000);
    }
  }, [alertHandlerContext]);
  return (
    <Alert
      alertMessage={alertHandlerContext.alertMessage}
      visible={alertHandlerContext.visible}
```
JavaScriptのReactでは、アッパーキャメルケースにするのはコンポーネントくらいかと思います。
通常のオブジェクトを入れた変数は、ローワーキャメルケースでお願いします。